DesignDoc.md

```diff
@@ -0,0 +1,157 @@
+# Design Document
+
+## _ReelDeal_ Design
+
+## 1. Problem Statement
+
+the reel deal is a letterboxd clone that lets you rate and review the movies you have recently seen to share your thoughts to your fellow friends and family. you can also preview the films your friends have left reviews on and add them to a watch list to later review. The reel deal will also feature popular movies if you're unsure what to watch next.
+
+## 2. Top Questions to Resolve in Review
+
+
+1. Working with multiple gsi.
+2. Figuring how to make a search function. (looking up certain movies by title, director or actor)
+3. Creating endpoint for movie API.
+4. Should I incorporate a follow feature for other movie buffs
+5. Find a way to link socials or sign in with socials 
+
+
+## 3. Use Cases
+
+_This is where we work backwards from the customer and define what our customers would like to do (and why). You may also include use cases for yourselves (as developers), or for the organization providing the product to customers._
```

Comment:
I like how you've written these (specifically that you've included a "so that I..." benefit of each story).

Reply:
We've asked for at least 9 stories ... 8 is close, and I think you've included a lot of functionality here, what's one or two more things that you can include here?